src/H5trace.c

```diff
@@ -120,8 +120,8 @@ H5_trace(const double *returning, const char *func, const char *type, ...)
     void *            vp                  = NULL;
     FILE *            out                 = H5_debug_g.trace;
     static hbool_t    is_first_invocation = TRUE;
-    H5_timer_t        function_timer      = {{0}, {0}, {0}, FALSE};
-    H5_timevals_t     function_times;
+    H5_timer_t        function_timer;
+    H5_timevals_t     function_times = {0.0, 0.0, 0.0};
```

Comment:
Hmm, I'm confused.  I thought they need to line up...

Reply:
They would if you swap 123 and 124 because 123 breaks the continuous lineup.

Reply:
Also, we let the formatter deal with this.